applications/ShapeOptimizationApplication/tests/algorithm_penalized_projection_test/materials_2D.json

```diff
@@ -0,0 +1,19 @@
+{
+    "properties": [{
+            "model_part_name": "Parts_solid",
+            "properties_id": 0,
+            "Material": {
+                    "name": "Shell_Material",
+                    "constitutive_law": {
+                            "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw"
```

Comment:
You can simply write `LinearElasticPlaneStress2DLaw`, the other stuff that comes beforehand is ignored

same for the variables

Reply:
when I do this, then I get the following error: "Please also provide the name of the application of constitutive law LinearElasticPlaneStress2DLaw"

Reply:
This is possible since #1455